scss/inc/_forms.scss

```diff
@@ -435,7 +435,7 @@ label {
         left: -10000px;
         top: 0;
 
-        [dir="rtl"] & {
+        body[dir="rtl"] & {
```

Comment:
**thought:** Not sure this is a correct approach. Are you sure the `dir` will be set only to the `body`? this does not allow having multi-level of `dir`.

Reply:
Please check the workaround that @meefox suggest to me. Teamwork! :)
I've checked crossbrowsing.